doc/topics/windows/windows-package-manager.rst

```diff
@@ -334,200 +339,232 @@ you need to install a specific version of a piece of software.
 
 Available parameters are as follows:
 
-:param str full_name: The Full Name for the software as shown in "Programs and
-    Features" in the control panel. You can also get this information by
-    installing the package manually and then running ``pkg.list_pkgs``. Here's
-    an example of the output from ``pkg.list_pkgs``:
+:param str full_name:
+    The Full Name for the software as shown in "Programs and Features" in the
+    control panel. You can also get this information by installing the package
+    manually and then running ``pkg.list_pkgs``. Here's an example of the output
```

Comment:
pkg.list_pkgs saltenv=NotARealEnv    this is like giving a raw data.

Reply:
Added this as a tip later on